src/main/java/org/opentripplanner/middleware/auth/Auth0Users.java

```diff
@@ -224,4 +236,33 @@ private static String getAuth0Url() {
             ? "http://locahost:8089"
             : "https://" + AUTH0_DOMAIN;
     }
+
+    /**
+     * Get an 0Auth token by using the Auth0 'Call Your API Using Resource Owner Password Flow' approach. Auth0 setup
+     * can be reviewed here: https://auth0.com/docs/flows/call-your-api-using-resource-owner-password-flow. If the user
+     * is successfully validated by Auth0 a bearer access token is returned, which is extracted and returned to the
+     * caller. In all other cases, null is returned.
+     */
+    public static String get0AuthToken(String username, String password) {
```

Comment:
Typo `get0AuthToken` should be `getAuth0Token`.

Reply:
Updated.